### ISSUE TITLE
Am i doing something wrong? % not moving.

### ISSUE THREAD

**Issue by rktkz** [Pure-L0G1C/Instagram#177]
This is what it looks like for me:
(I'm testing it on my own acc to see if it works then to actually put it to work)

[-] Wordlist: /root/wordlists/mykeys.txt
[-] Username: Rktkz
[-] Password: AlysonKurban1sm1n3.10
[-] Complete: 2.5641%
[-] Attempts: 1
[-] Browsers: 33
[-] Exists: True

first it was stuck @ 0% and it didn't say anything where it tells if the account exists or not
but it was going through the list looking like this:

[-] Wordlist: /root/wordlists/mykeys.txt
[-] Username: Rktkz
[-] Password: AlysonKurban1sm1n3.10
[-] Complete: 0.0%
[-] Attempts: 0
[-] Browsers: 33
[-] Exists:

(Different pass attempts)

now its stuck as shown b4 (First copy and paste)
Am I doing something wrong or is it tool not working currently?
I have used the tool months ago and it was the best tool to Bruteforce instagram accouns.
Please keep up your good work! (Take ur time and fix this tool) and reply to me whenever you have a chance to (:  +1:  .

**Comment by Pure-L0G1C** [comment]
No, you're not. Since this program is free and open to the public, everyone is using the same proxies.

**Comment by rktkz** [comment]
> No, you're not. Since this program is free and open to the public, everyone is using the same proxies.

So i just need to get a different proxy list.txt and rename it to the one you got or copy the proxies and paste it into the proxy list you made?
